src/util/misc.ts

```diff
@@ -46,27 +46,26 @@ export function touch(paths: string[] | string) {
   });
 }
 
-export function getFullUrl(webUrlHost: string, url: string, baseUrl: string) {
-  if (typeof url !== 'string' || !url) {
-    throw new Error(`Expected url to be a string, got [${url}] instead`);
-  }
+export function getFullUrl(url: string, baseUrl: string) {
   const urlObject = urlParser.parse(url, false, true);
   if (!urlObject.protocol) {
```

Comment:
We need a small commnet here to explain that there is nothing to do if we have a full URL already.